src/java/org/apache/cassandra/utils/MonotonicClock.java

```diff
@@ -75,9 +78,7 @@
 
         private static MonotonicClock precise()
         {
-            String sclock = System.getProperty("cassandra.clock");
-            if (sclock == null)
-                sclock = System.getProperty("cassandra.monotonic_clock.precise");
+            String sclock = System.getProperty("cassandra.monotonic_clock.precise");
```

Comment:
This is the only place that caused confusion (I'm not familiar with this part of the code), but taking into account the TODO above and my expectation that this class will be replace by `Clock` in further PRs, I'm good.